src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -108,25 +109,25 @@ public string SettingsPath
         /// <param name="logger">An ILogger implementation used for writing log messages.</param>
         public AnalysisService(string settingsPath, ILogger logger)
         {
-            this.logger = logger;
+            this._logger = logger;
 
             try
             {
                 this.SettingsPath = settingsPath;
 
-                scriptAnalyzerModuleInfo = FindPSScriptAnalyzerModule(logger);
-                var sessionState = InitialSessionState.CreateDefault2();
-                sessionState.ImportPSModulesFromPath(scriptAnalyzerModuleInfo.ModuleBase);
+                this._scriptAnalyzerModuleInfo = FindPSScriptAnalyzerModule(logger);
+                InitialSessionState sessionState = InitialSessionState.CreateDefault2();
+                sessionState.ImportPSModule(new [] { this._scriptAnalyzerModuleInfo.Path });
```

Comment:
We talked about this offline but I think it would be easier just to put `"PSScriptAnalyzer"` here instead of the path received from calling the `FindPSScriptAnalyzerModule(logger);` function. This also means that the `FindPSScriptAnalyzerModule` function can be totally removed from the code which is also a huge plus in my opinion.

I'd say give it a shot, if it doesn't work then this is fine.